apps/cardiovascular-health/tsconfig.app.json

```diff
@@ -0,0 +1,4 @@
+{
+  "extends": "./tsconfig.json",
+  "exclude": ["src/test.ts", "src/polyfills.ts", "src/**/*.spec.ts"]
```

Comment:
why are these excluding `src/polyfills.ts`? don't we want polyfills included in the app build?

Reply:
Okay, @semicolin  I will remove `src/polyfills.ts` from `tsconfig.app.json` for all `charts-on-fhir apps`.